packages/core/src/App/Containers/Layout/header/show-notifications.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { observer, useStore } from '@deriv/stores';
+import { Localize } from '@deriv/translations';
+import { ToggleNotifications } from 'App/Components/Layout/Header';
+
+const ShowNotifications = observer(() => {
+    const { notifications: notifications_store } = useStore();
+    const { is_notifications_visible, notifications, toggleNotificationsModal } = notifications_store;
+
+    return (
+        <div className='trading-hub-header__notification'>
+            <ToggleNotifications
+                count={notifications.length}
+                is_visible={is_notifications_visible}
+                toggleDialog={toggleNotificationsModal}
+                tooltip_message={<Localize i18n_default_text='View notifications' />}
+            />
```

Comment:
cant we use localize function?

Reply:
note: reverted to Localize based on latest info from yashim.